numpy/core/setup.py

```diff
@@ -81,7 +81,7 @@ def can_link_svml():
     if NPY_DISABLE_SVML:
         return False
     platform = sysconfig.get_platform()
-    return "x86_64" in platform and "linux" in platform
+    return "x86_64" in platform and "linux" in platform and sys.maxsize > 2147483647
```

Comment:
Line needs to be broken, I suggest something like:
```
    return ("x86_64" in platform
            and "linux" in platform
            and sys.maxsize > 2**31 - 1)
```
I'm a bit suspicious of the maxsize

Reply:
Thanks. I'll make this change. https://github.com/pypa/packaging/blob/main/packaging/tags.py#L40 uses the same test.

Reply:
The logic in the link is `sys.maxsize > 2**32`

Reply:
Well, `sys.maxsize` is `2**31 - 1` on 32 bit and `2**63 - 1` on 64 bit.
The tests are equivalent.